src/esp/gfx_batch/Renderer.cpp

```diff
@@ -1206,7 +1206,9 @@ void Renderer::draw(Mn::GL::AbstractFramebuffer& framebuffer) {
           .drawsSorted[i]
           /* Extract normal matrix */
           .setNormalMatrix(state_->absoluteTransformationsSorted[i]
-                               .transformationMatrix.normalMatrix())
+                               .transformationMatrix.rotationScaling()
+                               .inverted()
+                               .transposed())
```

Comment:
This addresses the erroneous normalMatrix() but does not yet address flipping the backFace culling winding value.

Reply:
Fixing this for batch rendering is outside the scope of this PR.